numpy/random/src/pcg64/pcg64.h

```diff
@@ -207,6 +209,13 @@ static inline uint64_t pcg_output_xsl_rr_128_64(pcg128_t state) {
                      state >> 122u);
 }
 
+static inline uint64_t
```

Comment:
Not a bug deal, but should these files be NumPy-ified? e.g. inline->NPY_INLINE ?  I think everything supports plain old inline now, so not strictly necessary.

Reply:
Yes, now that we require C99, inline should be fine even though it violates my sense of consistency :) At some point we should probably replace all the `NPY_INLINE` macros with `inline` just to make it a bit easier for new contributors who may know C but not NPY.